nohost/src/NoHostServer.js

```diff
@@ -42,9 +43,22 @@ define(function (require, exports, module) {
      * instrumentation is enabled
      */
     NoHostServer.prototype.add = function (liveDocument) {
+        var id = liveDocument.doc.language.getId();
+
         if (liveDocument.setInstrumentationEnabled) {
-            // enable instrumentation
-            liveDocument.setInstrumentationEnabled(true);
+            //If new HTML document is empty, inject defaultHTML file
+            //so live preview doesn't break
+            if(liveDocument.doc.getText() === "" && id === "html") {
+                liveDocument.doc.setText(defaultHTML);
```

Comment:
Rather, what problem is this solving? Why do we have to inject anything?

If we can't avoid it, we should be inserting whatever thimble provides us, or falling back to the `defaultHTML`.